### ISSUE TITLE
No PC-Mask part code没有PC-Mask的部分代码

### ISSUE THREAD

**Issue by tjtlucid** [a2502503/EchoONE#4]
The code you provided, including the testing code and training code, requires preloading files of cosine similarity and clustering results from your own training set, which is the part PC Mask module mentioned in the paper. Can you provide the relevant code.
您所提供的代码中，包括测试代码和训练代码，都需要预先加载余弦相似性和您自己训练集中的聚类结果的文件，也就是论文中提到的部分PC-Mask模块，您可以提供相关的代码吗。

**Comment by wps278** [comment]
同求

**Comment by 2399252807** [comment]
同求
